src/utils.js

```diff
@@ -166,21 +166,27 @@ export function isThisYear(date: Date) {
 }
 
 // eslint-disable-next-line flowtype/no-weak-types
-export function makeRelativeFormatter(options: any): () => ?any {
-  let format
-  return function() {
-    if (format) return format
-    if ('Intl' in window && 'RelativeTimeFormat' in window.Intl) {
-      try {
-        // eslint-disable-next-line flowtype/no-flow-fix-me-comments
-        // $FlowFixMe: missing RelativeTimeFormat type
-        format = new Intl.RelativeTimeFormat(undefined, options)
-        return format
-      } catch (e) {
-        if (!(e instanceof RangeError)) {
-          throw e
-        }
+export function makeRelativeFormat(locale: string, options: any): ?any {
```

Comment:
I understand `any` wasn't added or changed in this PR, but could we get rid of it here? We could define a type that declares the specific parameters that we support and forward to `Intl.RelativeTimeFormat`, and return a type that basically just implements the `format()` method (since that's the only method that we interface with).

We can later drop all of that when Flow implements `Intl.RelativeTimeFormat` natively.